packages/db/src/controller/lmdb.ts

```diff
@@ -0,0 +1,68 @@
+import {Logger} from "@lodestar/utils";
+import {DatabaseController, DatabaseOptions, DbReqOpts, FilterOptions, KeyValue} from "./interface.ts";
+import {LevelDbControllerMetrics} from "./metrics.ts";
+
+export type LmdbControllerModules = {
+  logger: Logger;
+  metrics?: LevelDbControllerMetrics | null;
+};
+
+export class LmdbController implements DatabaseController<Uint8Array, Uint8Array> {
```

Comment:
This code must be covered with unit tests. May be using same fixture to test different implementations of db controllers. 

It is important as if something missed goes wrong here make it very difficult to debug on higher layers.